test/test_config.h.in

```diff
@@ -1,10 +1,126 @@
-/* THIS HEADER WAS AUTOMATICALLY GENERATED 
+/* THIS HEADER WAS AUTOMATICALLY GENERATED
  * AND SHOULD NOT BE DIRECTLY MODIFIED.
  *
  * See test/test_config.h.in in the ignition-common
  * repo for the original source of this header.
 */
 
-#define PROJECT_BINARY_PATH "${PROJECT_BINARY_DIR}"
-#define PROJECT_SOURCE_PATH "${PROJECT_SOURCE_DIR}"
-#define TEST_PATH "${PROJECT_SOURCE_DIR}/test"
+#ifndef IGNITION_TESTING__TEST_CONFIG_H_
+#define IGNITION_TESTING__TEST_CONFIG_H_
+
+#include <ignition/common/Filesystem.hh>
+#include <ignition/common/Util.hh>
+
+namespace ignition
+{
+  namespace testing
```

Comment:
This namespace looks like it could cause conflicts in the future. Is there a reason not to use `common`?

Reply:
And after I wrote this comment I saw that there's a precedent in `util.hh`. I'm still not convinced this is a good idea, but I'd be ok leaving as is because it's following an existing pattern.

Reply:
I'm fine with changing it to `ignition::common::testing`.

Like you said, it was following the precedent already there to some degree.